src/native/managed/cdacreader/tests/CodeVersionsTests.cs

```diff
@@ -254,6 +254,24 @@ internal static Target CreateTarget(
         return target;
     }
 
+    internal static Target CreateTarget(
+        MockTarget.Architecture arch,
+        MockCodeVersions builder,
+        Mock<IRuntimeTypeSystem> mockRuntimeTypeSystem)
+    {
+        TestPlaceholderTarget target = new TestPlaceholderTarget(
+            arch,
+            builder.Builder.GetReadContext().ReadFromTarget,
+            builder.Types);
+
+        IContractFactory<ICodeVersions> cvfactory = new CodeVersionsFactory();
+        ContractRegistry reg = Mock.Of<ContractRegistry>(
+            c => c.CodeVersions == cvfactory.CreateContract(target, 1)
+                && c.RuntimeTypeSystem == mockRuntimeTypeSystem.Object);
+        target.SetContracts(reg);
+        return target;
+    }
```

Comment:
Previous CodeVersion tests use custom "mock" classes. Now that we have `Moq` integrated, I used it to create the new GCStressCodeCopy tests.